client/packages/system/src/Stock/api/hooks/keys.ts

```diff
@@ -1 +1,2 @@
 export const STOCK_LINE = 'stock_line';
+export const LEDGER = 'ledger';
```

Comment:
Any reason to separate those keys ? We would have to remember to invalidate both ?

Reply:
Changed it